Rename the CPACSCreator to TiGLCreator

svengoldberg opened the pull request:
## Description
Before merging the [CPACSCreator-branch](https://github.com/DLR-SC/tigl/tree/cpacs_creator_dev_merge) back into the master, the program should be renamed to TiGLCreator. This also includes the change of many file names and file contents.

## Checks
I did some obvious tests (configuring, building and starting the executable and doing some random stuff).

Towards the reviewers: Probably you can think of other tests or something that I missed. There are a lot of changes ....

Fixes #1137 

## Checklist:
| Task                                                    | Finished                                                  | Reviewer Approved          |
|---------------------------------------------------------|-----------------------------------------------------------|----------------------------|
| At least one test for the new functionality was added.  | <ul><li>- [ ] yes </li><li>- [x] does not apply</li></ul> | <ul><li>- [ ] OK</li></ul> |
| New classes have been added to the Python interface.    | <ul><li>- [ ] yes </li><li>- [x] does not apply</li></ul> | <ul><li>- [ ] OK</li></ul> |
| The code is properly documented with doxygen docstrings | <ul><li>- [ ] yes </li><li>- [x] does not apply</li></ul> | <ul><li>- [ ] OK</li></ul> |
| Changes are documented at the top of ChangeLog.md       | <ul><li>- [x] yes </li><li>- [ ] does not apply</li></ul> | <ul><li>- [ ] OK</li></ul> |

merakulix commented on tests/tiglcreator-report/README.md:
The executables name "tiglviewer-3" got replaced by just "tiglcreator"  did I see that right?

merakulix commented on tests/tiglcreator-report/generate_report.py:
```diff
@@ -123,17 +123,17 @@ def add_screenshot_command(content, cpacs_dict, screenshot_path, view=None):
 def create_screenshots(tigl_script):
     """
-    This calls tiglviewer-3 with the provided tigl script
+    This calls tiglcreator-3 with the provided tigl script
```
again, remove "-3"?

merakulix commented on tests/tiglcreator-report/generate_report.py:
```diff
@@ -276,12 +276,12 @@ def main(input_files, output_file, update, tigl_script, screenshot_dir, download
                              screenshot_dir)
 
-    # take the screenshots using tiglviewer-3
+    # take the screenshots using tiglcreator-3
```
I didn't mark all the lines in which it occurs, I suppose all the "-3" this script need to be removed.

merakulix commented on CMakeLists.txt:
```diff
@@ -186,23 +186,23 @@ endif()
 if (APPLE)
   set(CPACK_PACKAGE_EXECUTABLES
-    "CPACS-Creator" "CPACS Creator"
+    "TIGL-Creator" "TiGL Creator"
```
Probably not that important but should it either be "TIGL" or "TiGL"?

merakulix reviewed (commented):
This was truely a lot of iterative rewording :) I built and ran tests, I checked you're changes in code and GUI, and I only found one python skript that will probably not work anymore because auf renaming the executable, see in comments. 
Everything else seem to work.
But did you check the python bindings and ran some python tests, see if classname changes didn't produce any errors there?

svengoldberg commented:
> Could you please verify, that in the TiGL console you still have `app.viewer` and not `app.creator`? Thanks!

@joergbrech Yes, the TiGL console does exclusively recognoize `app.viewer`, but not `app.creator`. Only the shown text was wrong, but I fixed it.

I also fixed the mentioned missing CamelCase at several places and even found some more inconsistent naming
